### PR TITLE
i18n(fr): update `recipes/modified-time.mdx`

### PR DESCRIPTION
**Pull request by ArmandPhilippot**
#### Description (required)

Translate the note added in #9520 in the French translation of `recipes/modified-time.mdx`.

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/recipes/modified-time.mdx [hunk 1]
```diff
@@ -9,6 +9,10 @@ import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
 Apprenez à construire un [plugin remark](https://github.com/remarkjs/remark) qui ajoute l'heure de la dernière modification au Front Matter de vos fichiers Markdown et MDX. Utilisez cette propriété pour afficher l'heure de modification dans vos pages.
 
+:::note[Utiliser l'historique Git]
+Cette recette calcule le temps en fonction de l'historique Git de votre dépôt et celui-ci peut ne pas être précis sur certaines plateformes de déploiement. Votre hébergeur peut effectuer des **clones superficiels** qui ne récupèrent pas l'historique Git complet.
```
Comment (pioupia) [inline src/content/docs/fr/recipes/modified-time.mdx:13]:
```suggestion
Cette recette calcule la date en fonction de l'historique Git de votre dépôt, et celle-ci peut ne pas être précise sur certaines plateformes de déploiement. Votre hébergeur peut effectuer des **clones superficiels** qui ne récupèrent pas l'historique Git complet.
```

**Review by pioupia — COMMENTED** [review]
For me, it's better to say “the date” in this context rather than “the time”. Time doesn't have a good primary definition for the purpose of the page.

**Review by thomasbnt — APPROVED** [review]
LGTM!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**